src/libstd/ffi/c_str.rs

```diff
@@ -455,6 +455,20 @@ impl From<NulError> for io::Error {
     }
 }
 
+#[unstable(feature = "frombyteswithnulerror_impls", reason = "recently added", issue = "39925")]
```

Comment:
Oh unfortunately impls aren't subject to stability the same way as other parts of the standard library, so these'll be "insta-stable" so it's ok to just go ahead and tag this with `#[stable]`